### PR TITLE
Fix: make isLimitReached more coherent

### PR DESCRIPTION
**Pull request by TheDZhon**
- [x] Update the `isLimitReached()` impl
- [x] Add test case for [`TokenRebaseLimiter`](https://github.com/lidofinance/lido-dao/blob/fix/rebaseLimiter-isLimitReached/test/0.8.9/positive-token-rebase-limiter.test.js#L230-L244)
- [x] Add test case for [`OracleReportSanityChecker`](https://github.com/lidofinance/lido-dao/blob/fix/rebaseLimiter-isLimitReached/test/0.8.9/oracle-report-sanity-checker.test.js#L933-L959)

### CODE REVIEW

**Review by folkyatina — APPROVED** [review]
Looks sane

**Review by loga4 — APPROVED** [review]
lgtm

**Review by Psirex — APPROVED** [review]
LGTM👍🏻

File: contracts/0.8.9/lib/PositiveTokenRebaseLimiter.sol [hunk 1]
```diff
@@ -102,16 +102,11 @@ library PositiveTokenRebaseLimiter {
...
         if (_limiterState.positiveRebaseLimit == UNLIMITED_REBASE) return false;
-        if (_limiterState.currentTotalPooledEther < _limiterState.preTotalPooledEther) return false;
 
-        uint256 accumulatedEther = _limiterState.currentTotalPooledEther - _limiterState.preTotalPooledEther;
-        uint256 accumulatedRebase;
-
-        if (_limiterState.preTotalPooledEther > 0) {
-            accumulatedRebase = accumulatedEther * LIMITER_PRECISION_BASE / _limiterState.preTotalPooledEther;
-        }
+        uint256 maxTotalPooledEther = _limiterState.preTotalPooledEther +
```
[the remark this answers is not in the document]
Comment (TheDZhon) [reply contracts/0.8.9/lib/PositiveTokenRebaseLimiter.sol:106]: totally makes sense, ty!
decided to proceed with your idea :+1:

**Pull request merged**